install/docker/opensuse-docker-install.md

```diff
@@ -5,13 +5,24 @@
 > - This guide assumes you have
 >   [already installed the Kata Containers packages](../opensuse-installation-guide.md).
 
-1. Install the latest version of Docker with the following commands:
-
-   > **Note:** This step is only required if Docker is not installed on the system.
+1. Install Docker with the following commands:
+
+   > **Notes:**
+   >
+   > - This step is only required if Docker is not installed on the system.
+   > - Newer versions of Docker have
+   >   [removed devicemapper support](https://github.com/kata-containers/documentation/issues/373)
+   >   so the commands below install the latest version which includes
+   >   devicemapper support.
+   > - To remove the lock on the docker package to allow it to be updated:
+   >   ```sh
+   >   $ sudo zypper removelock docker
+   >   ```
 
    ```bash
    $ sudo zypper -n install libcgroup1
-   $ sudo zypper -n install docker
+   $ sudo zypper -n install -f docker-18.06.1_ce-51.1
```

Comment:
Another neat way to do this without hard coding a specific docker version is:

`docker -n install docker<18.09'`

This is going to work on SLES too.

Reply:
Thanks @marcov - I've updated the branch for opensuse + sles.